src/DacpacTool/Program.cs

```diff
@@ -132,6 +142,33 @@ private static int BuildDacpac(BuildOptions options)
             return 0;
         }
 
+        private static int InspectIncludes(InspectOptions options)
+        {
+            // Wait for a debugger to attach
+            WaitForDebuggerToAttach(options.Debug);
```

Comment:
I guess we can refactor this method to accept a `BaseOptions` instance now and just pass in the options here.

Reply:
This got implemented as well.